SPECS-EXTENDED/xdp-tools/xdp-tools.spec

```
@@ -0,0 +1,157 @@
+Summary:        Utilities and example programs for use with XDP
+Name:           xdp-tools
+Version:        1.2.9
+Release:        1%{?dist}
+License:        GPLv2, LGPLv2, BSD
```

We're moving towards using [SPDX expressions](https://spdx.org/licenses/) for our license tags. I went ahead and double-checked that the licenses present in this repo correspond to the following expression:

```suggestion
License:        GPL-2.0-only AND LGPL-2.1-only AND BSD-2-Clause
```

---

done, thanks!